main.py

```diff
@@ -0,0 +1,21 @@
+from googleSheetsService import wks
+class Person:
+    def __init__(self,name,lastName,age,height,weight):
+        self.name = name
+        self.lastName = lastName
+        self.age = age
+        self.height = height
+        self.weight = weight
+
+def create_user():
+    name = input("Enter your First Name:")
```

Comment:
create a user service and move this function there, it should return user object